dashboards-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -681,7 +727,43 @@ export function ReportSettings(props: ReportSettingProps) {
       </div>
     );
 
+    const displayNotebooksSelect = 
+    reportSourceId === 'notebooksReportSource' ? (
+      (
```

Comment:
why two parentheses?

Reply:
No reason, good catch. I can remove the inner set